ui/appui-react/src/appui-react/widgets/WidgetDef.tsx

```diff
@@ -537,16 +541,97 @@ export class WidgetDef {
    * I.e. opens the stage panel or brings the floating widget to front of the screen.
    * @public
    */
-  public show() {
+  public async show() {
     const frontstageDef = UiFramework.frontstages.activeFrontstageDef;
     const state = frontstageDef?.nineZoneState;
     if (!state) return;
     if (!frontstageDef.findWidgetDef(this.id)) return;
 
-    frontstageDef.dispatch({
-      type: "WIDGET_TAB_SHOW",
-      id: this.id,
-    });
+    const tabLocation = getTabLocation(state, this.id);
+    if (tabLocation && isPopoutTabLocation(tabLocation)) {
+      const testWindow = UiFramework.childWindows.find(
+        tabLocation.popoutWidgetId
+      );
+      if (testWindow) {
+        // We have to check if Safari is the browser being used. Safari has security
+        // measures that do not allow you to change focus to another tab programmatically.
+        // Therefore the only and recommended way around this at the moment, is to close
+        // and then open the tab again. This remounts the popout. Thats why this will not be done,
+        // because we don't want the state of the widget to be lost. Instead a toast will
+        // be displayed that informs the user that safari cannot focus the popout.
+        // Apple support docs: https://discussions.apple.com/thread/251676767?sortBy=best
+        const isSafari =
+          navigator.userAgent.toLowerCase().indexOf("safari/") > -1;
+
+        const isChrome =
+          navigator.userAgent.toLowerCase().indexOf("chrome") > -1;
+
+        const hideFocusFail = await UiFramework.getUiStateStorage().getSetting(
+          "popoutFocus",
+          "hidePopoutFocusFail"
+        );
+        if (!isChrome && isSafari && hideFocusFail.setting !== true) {
```

Comment:
Getting the setting is only needed if the 2 first conditions are true, and if the setting is true we should probably simply do nothing, instead of trying the focus anyway, in case this starts to throw an error at some point.

Reply:
Reworked to this logic